### DISCUSSION TITLE
Upload embedded excalidraw files

### DISCUSSION THREAD

**Discussion by weihong0827** [ObsidianPublisher/obsidian-github-publisher#224] [General]
When uploading with GitHub publisher, it seems that normal image files and pdf is uploaded successfully but not the excalidraw files. Is there a way to fix this
<img width="813" alt="image" src="https://github.com/ObsidianPublisher/obsidian-github-publisher/assets/48542562/e98f76b4-1ced-4941-9c95-bbf915b5d6c4">

**Comment by Mara-Li** [comment]
Because I don't support `.excalidraw`. You need to use the SVG parameters for embedding image
